CMakeLists.txt

```diff
@@ -321,8 +321,11 @@ if(webgui)
 endif()
 
 foreach(dir_to_copy ${directories_to_copy})
-  # For target dependencies:
+  # Create a list of files for which we want dependency tracking.
+  # Exclude certain endings, so CMake doesn't complain when they are deleted (JIRA#9688)
   file(GLOB_RECURSE artifacts_in ${dir_to_copy}/*)
+  list(FILTER artifacts_in EXCLUDE REGEX "\.(d$|swp|so|dylib|bak|old)$")
```

Comment:
OK, but what about `dll`, `exp`, `pcm` , `lib` , ... ?

Reply:
As we just discussed, it's mostly for cosmetics to keep the dependencies cleaner. If you don't `dir` into the source tree and run from there, you should not see this issue.

Reply:
On the other hand if you list `|so|dylib` there is no reason not to list `|lib|dll`.  I.e. they are either all needed or all not needed :).